### DISCUSSION TITLE
Support for category fields with multiple values

### DISCUSSION THREAD

**Discussion by miteshyadav** [awslabs/gluonts#1477] [Q&A] [answered]
## Description

Is there a functionality to incorporate many- to-many (multi-label) categorical feature in GLuonTS? It would be helpful to have a feature like that. Currently it just supports one to one mapping. FOr example, a product may belong to multiple categories and not just one

**Comment by lostella** [comment, marked as answer]
@miteshyadav for the models that support the `feat_static_cat` feature (eg DeepAR) that's already the case: you just need to put there an array of length > 1.

I'm moving this into the discussions, if you don't mind, so that others can find this answer.

**Reply by pratikgehlott** [reply]
hey, @lostella  when I am using `feat_static_cat`, I am getting an error `Array 'feat_static_cat' has a bad shape - expected 1 dimension, got 2. `.

I am giving my `feat_static_cat` as -

'feat_static_cat' :[
          df.f1[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f2[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f3[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f4[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f5[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f6[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f7[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f8[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f9[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f10[:pd.to_datetime('2018-12-31 00:00:00')],
          df.f11[:pd.to_datetime('2018-12-31 00:00:00')],
      ]
These are all my categorical features.
Is this right? or should I do something different
